spec/controllers/api/irs_attempts_api_controller_spec.rb

```diff
@@ -201,9 +204,21 @@
         'IRS Attempt API: Events submitted',
         rendered_event_count: existing_events.count,
         authenticated: true,
+        elapsed_time: 0.1234,
         success: true,
         timestamp: time.iso8601,
       )
+
+      #      expect(@analytics).to receive(:track_event).with(
+      #        'IRS Attempt API: Events submitted',
+      #        {
+      #          rendered_event_count: existing_events.count,
+      #          authenticated: true,
+      #          elapsed_time: 0.1234,
+      #          success: true,
+      #          timestamp: time.iso8601,
+      #        }
+      #      )
```

Comment:
We can remove this now.

![delete](https://media.giphy.com/media/P7PmvHY6kzAqY/giphy.gif)